### DISCUSSION TITLE
Cannot create GrpcChannel in Uno Platform WebAssembly (System.NullReferenceException)

### DISCUSSION THREAD

**Discussion by t-sorger** [grpc/grpc-web#1253] [Q&A - General]
I am trying to do a sample of *Uno Platform* using *gRPC-Web*.

Therefore I got inspired by [the content provided by this article](https://azure.github.io/AppService/2021/03/15/How-to-use-gRPC-Web-with-Blazor-WebAssembly-on-App-Service.html). I followed the instructions of the article and created a *BlazorApp* which used a weather service with *gRPC-Web*. After that I also included an other service, the counter service, as seen in [this gRPC-Web example](https://github.com/grpc/grpc-dotnet/tree/master/examples/Counter).

After everthing was working, I added a *Uno Platform WebAssembly* app to replace the *BlazorApp* client.

##### Problem:
When trying to create a ``` GrpcChannel ``` the application gets a ``` System.NullReferenceException ```.

The code snippet for creating the channel looks like this:
```
var baseUri = "https://localhost:44366";
var channel = GrpcChannel.ForAddress(baseUri, new GrpcChannelOptions());
```

This is the same code as used in the *BlazorApp* sample.

The sample code for the *BlazorApp* and the *Uno Platform WebAssembly* can be found in [this repository](https://github.com/t-sorger/UnoGrpcWebAssemblyDemo).

Any idea/suggestion/help on how to fix this would be appreciated.
([Question on stackoverflow](https://stackoverflow.com/questions/72573187/cannot-create-grpcchannel-in-uno-platform-webassembly-system-nullreferenceexcep))

**Comment by sampajano** [comment]
Thanks for the info! I'm guessing the error happened during server setup, but not specifically wrt to the grpc-web JS client? 😃
